src/Spout/Reader/Helper/XLSX/SharedStringsHelper.php

```diff
@@ -141,11 +144,19 @@ public function extractSharedStrings()
 
             $unescapedTextValue = $escaper->unescape($textValue);
 
-            // The shared string retrieval logic expects each cell data to be on one line only
-            // Encoding the line feed character allows to preserve this assumption
-            $lineFeedEncodedTextValue = $this->escapeLineFeed($unescapedTextValue);
-
-            $this->writeSharedStringToTempFile($lineFeedEncodedTextValue, $sharedStringIndex);
+            if (self::$KEEP_ALL_STRINGS_IN_MEMORY) {
+                if (!Isset($this->inMemoryContents)) {
```

Comment:
Isset => isset. I think you don't even need this check. PHP will accept you setting the array value directly